app/models/pageflow/entry.rb

```diff
@@ -43,6 +43,7 @@ class PasswordMissingError < StandardError
       create_draft!(home_button_enabled: theming.home_button_enabled_by_default)
       draft.storylines.create!(configuration: {main: true})
       theming.widgets.copy_all_to(draft)
+      theming.copy_defaults_to(draft)
```

Comment:
What do you think about also moving the `widgets.copy_all_to` call to the new `copy_defaults_to` method.